### DISCUSSION TITLE
Consider renaming repository (back) to nord-tmux due to "tmux" being the tpm plugin name

### DISCUSSION THREAD

**Discussion by ericvw** [arcticicestudio/nord#188] [Suggestions & Ideas]
Regarding the migration mentioned in nordtheme/tmux#78, the result of the installation is the plugin named `tmux` because the tmux plugin manager only uses the repository name as the plugin name.

Output from `prefix` + `I` below:

```
Already installed "tpm"                                                                        [0/0]
Installing "tmux"
  "tmux" download success

TMUX environment reloaded.

Done, press ENTER to continue.
```

Could the repository be renamed back to `nord-tmux`; or `tmux-nord` to keep the tool name to the left to be closer in consistency with the names of the other repositories?

As an aside, I just started porting all my tools to Nord as I have recently stumbled across this great color palette and associated themes — thank you for all of this and the great post in nordtheme/tmux#78 that outlines the future of Nord!

**Comment by svengreb** [comment]
Hi @ericvw :wave:, thanks for your contribution :+1:
Nice to see that you like Nord 😄

In this case the problem must be fixed by the upstream which is `tpm` is this case. The format of `OWNER/REPOSITORY` works fine and the plugin is installed successfully, but the only unpleasant side effect is the output of the installation command which only uses the `REPOSITORY` part of this format instead of the full identifier.
Renaming the repository is not an option because one goal of the migration into this organization is the fact that repositories can now be named in a “cleaner“ way without the need to use prefixes and with a self-explanatory project context through the `nordtheme` namespace.

I‘ve quickly searched the `tpm` repository and found https://github.com/tmux-plugins/tpm/issues/230 which is already covered by https://github.com/tmux-plugins/tpm/pull/217. As soon as this is merged the problem of the output should work fine and the actual structure of the plugins on disk should also be cleaner. I‘ve upvoted and subscribed to the PR to see when this gets merged.

In the meantime, if you‘d like to “fix“ this for your setup you can also [install the Nord plugin manually like described in the port documentations][1]. Please note that they have not been updated to the new structure and migrations yet so the paths and the plugin name must be changed accordingly.

[1]: https://www.nordtheme.com/docs/ports/tmux/installation#manual

**Reply by ericvw** [reply]
@svengreb, thank you for taking the time to reply and providing a thorough response.

I'll redirect my attention to [tmux-plugins/tpm](https://github.com/tmux-plugins/tpm) and explore how I can contribute further over there!

**Reply by svengreb** [reply]
I‘ll leave this discussion open so that it is easier for others to find it too and to also keep track of this problem 😄
